edx/frontend-app-discussions
# Add post banners [BD-38] [BB-4737] [TNL-8655]

## Description

This PR Implement post banners for:

- Pinned posts
- Endorsed posts
- Reported posts
- closed posts

## Supporting Information

[BB-4737](https://tasks.opencraft.com/browse/BB-4737)
[TNL-8655](https://openedx.atlassian.net/browse/TNL-8655)

## Testing Instructions

- Checkout to this branch `tinumide/add_post_banners` in this repository 
- Login as a Discussions Moderator user
- To create a Discussions Moderator user visit the instructor tab as staff here: http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/instructor#view-membership (for the demo course) and in the membership tab, add the user to a role under the "Course Team Management" section.
- Verify that banners are on posts that have been `Pinned`, `Endorsed`, `Reported` or `Closed` as seen in the mockup [here](https://www.figma.com/file/LVFnO9oRHYiamoicZuXDAC/Discussion-page?node-id=1840%3A35281)
- Login as a Learner
- Verify that banners are on posts that have been `Pinned`
